webview/src/App.jsx

```diff
@@ -8,6 +8,13 @@ function App() {
   const [error, setError] = React.useState(false);
   const [loading, setLoading] = React.useState(true);
 
+  const toolbarOptions = {
```

Comment:
hey @SajidAlamQB  this looks like it should be inside a config or something similar if we have one in the vscode?

Reply:
I don't think we do, most of the viz options and settings are found in this file for the extension.